tests/baselines/reference/bluebirdStaticThis.types

```diff
@@ -27,7 +27,7 @@ export declare class Promise<R> implements Promise.Thenable<R> {
 
     static try<R>(dit: typeof Promise, fn: () => Promise.Thenable<R>, args?: any[], ctx?: any): Promise<R>;
 >try : { <R>(dit: typeof Promise, fn: () => Promise.Thenable<R>, args?: any[], ctx?: any): Promise<R>; <R_1>(dit: typeof Promise, fn: () => R_1, args?: any[], ctx?: any): Promise<R_1>; }
->    : ^^^ ^^   ^^              ^^  ^^                         ^^    ^^^     ^^   ^^^   ^^^          ^^^   ^^   ^^              ^^  ^^         ^^    ^^^     ^^   ^^^   ^^^^^^^^^^^^^^^^^^
+>    : ^^^ ^^   ^^              ^^  ^^                         ^^    ^^^     ^^   ^^^   ^^^          ^^^^^^^^   ^^              ^^  ^^^^^^^^^^^^^    ^^^     ^^   ^^^   ^^^^^^^^^^^^^^^^^^

```

Comment:
Another case where reuse is seemingly lost but probably wrong before (I'll stop commenting on this class of change).